torch/__init__.py

```diff
@@ -2065,6 +2065,28 @@ def _constrain_as_size(symbol, min: Optional[builtins.int] = None, max: Optional
     """
     torch.sym_constrain_range_for_size(symbol, min=min, max=max)
 
+def import_device_backends():
+    if sys.version_info < (3, 10):
+        from importlib_metadata import entry_points
+    else:
+        from importlib.metadata import entry_points
+
+    for backend in entry_points(group='torch.backends'):
+        try:
+            backend_hook = backend.load() 
+            if not hasattr(backend_hook, 'init_custom_backend'):
```

Comment:
Please change a name of function to init_device_backend for consistency

Reply:
Why don't we use the official `load()` function of `EntryPoint` to get the init function to call?

Reply:
Sure, we can simplify to initialize at load